client/src/components/TopBar.jsx

```diff
@@ -15,28 +16,36 @@ const TopBar = () => {
       mx: 1.5,
     },
 
-    logoLink: {
-      px: 1,
-    },
-
     buttonsBox: {
       display: "flex",
       width: "100vw",
       justifyContent: "end",
     },
+
+    titleContainer: {
+      display: "flex",
+      backgroundColor: "#FFE3BE",
+      height: 60,
+      borderRadius: 9,
+      boxShadow: 3,
+      pl: 2,
+      pr: 4,
+      ml: 2,
+    },
   };
 
   return (
     <>
       <Box sx={styles.topBarContainer}>
-        <Link
-          href="/"
-          variant="h3"
-          color="primary.contrastText"
-          underline="hover"
-          sx={styles.logoLink}
-        >
-          TableBook
+        <Link href="/" underline="hover" color="primary.contrastText">
+          <Box sx={styles.titleContainer}>
+            <img
+              src="/favicon.png"
+              alt="image failed to load"
+              style={styles.image}
```

Comment:
styles.image doesnt seem to exist. You can add sx styles to image component like this as MUI doens't have an Image component: 
https://stackoverflow.com/questions/61263669/does-material-ui-have-an-image-component
(top answer)